experimental/pgbouncer/helmfile.yaml

```diff
@@ -0,0 +1,15 @@
+# generate a transiant postgres-xl db. This db will 
+# NOT be able to withstand restarts.
+releases:
+  - name: zav-db-tran
```

Comment:
Can you change this to a more descriptive name?

Reply:
ok done